Hyper/Controllers/Bots/Spawn/StandardBotSpawnStrategy.cs

```diff
@@ -12,25 +12,24 @@ internal class StandardBotSpawnStrategy : AbstractBotSpawnStrategy
 
     private readonly int _maxBots;
 
-    private readonly int _minSpawnRadius;
+    private readonly float _minSpawnRadius;
 
-    private readonly int _maxSpawnRadius;
+    private readonly float _maxSpawnRadius;
 
     public StandardBotSpawnStrategy(Scene scene, Settings settings) : base(scene, settings)
     {
-        _maxBots = 10 * settings.RenderDistance * settings.RenderDistance * Chunk.Size / 32 * Chunk.Size / 32;
-        _minSpawnRadius = Chunk.Size * settings.RenderDistance / 3;
-        _maxSpawnRadius = Chunk.Size * settings.RenderDistance * 2 / 3;
+        _maxBots = 4 * settings.RenderDistance * settings.RenderDistance * Chunk.Size / 32 * Chunk.Size / 32;
+        _minSpawnRadius = Chunk.Size * settings.RenderDistance * 0.5f;
+        _maxSpawnRadius = Chunk.Size * settings.RenderDistance * 1.6f;
```

Comment:
doesn't this mean that bots can spawn outside of the generated terrain?

Reply:
It didn't break when I tested but that may have been blind luck. I'll change max to 1 (better safe than sorry) and min to 0.7 to give the player some chance to escape